src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1009,6 +1010,15 @@ public FireworkEffect get(FireworkExplodeEvent e) {
 				}
 			}, 0);
 		}
+		//PlayerRiptideEvent
+		if (Skript.classExists("org.bukkit.event.player.PlayerRiptideEvent")) {
+			EventValues.registerEventValue(PlayerRiptideEvent.class, ItemStack.class, new Getter<ItemStack, PlayerRiptideEvent>() {
```

Comment:
This should be ItemType, not ItemStack. I am aware there's a lot of event values using ItemStack but that's going to change at some point so I'd rather start doing it now.